functional/cypress-tests/cypress/integration/dashboards.js

```diff
@@ -1,175 +1,190 @@
-          "Title length"
-        ).should("is.visible");
-      } else {
-        cy.get('[data-cy="board-form-title-input-error"]').should(
-          "not.visible"
-        );
-      }
-    });
-  });
-
-  columnEdgeValues.forEach(value => {
-    it(` For Columns input is displayed and submit is impossible for incorrect values. Edge value : ${value}`, () => {
-      cy.addDashboard(dashboardNameGen(), value, "10", true);
-      if (value == "3" || value == "21") {
-        cy.get('[data-cy="board-form-columns-input-error"]').should(
-          "is.visible"
-        );
-      } else {
-        cy.get('[data-cy="board-form-columns-input-error"]').should(
-          "not.visible"
-        );
-      }
-    });
-  });
-
-  switchIntervalEdgeValues.forEach(value => {
-    it(` For Switch Interval input is displayed and submit is impossible for incorrect values. Edge value: ${value}`, () => {
-      cy.addDashboard(dashboardNameGen(), "8", value, true);
-      if (value == "2") {
-        cy.get('[data-cy="board-form-switch-interval-input"]').should(
-          "is.visible"
-        );
-      } else {
-        cy.get('[data-cy="board-form-switch-interval-input"]').should(
-          "not.visible"
-        );
-      }
-    });
-  });
-});
-
-describe("Dashboard switcher", () => {
-  beforeEach(() => {
-    cy.visit("/");
-    cy.login();
-  });
-
-  it("Manual switching works", () => {
-    const dashboardName = dashboardNameGen();
-    let prevDashboardName, nextDashboardName;
-
-    cy.addDashboard(dashboardName, "8", "3");
-    cy.chooseDashboard(dashboardName);
-    cy.contains('[data-cy="navbar-title-header"]', `${dashboardName}`)
-      .should('is.visible');
-    prevDashboardName = cy.get('[data-cy="previous-board-button"]')
-      .invoke('attr', 'title')
-      .then(title => {
-        prevDashboardName = title.toString();
-        cy.get('[data-cy="previous-board-button"]')
-          .click();
-        cy.contains('[data-cy="navbar-title-header"]', `${prevDashboardName}`)
-          .should("is.visible");
-      })
-      .toString();
-    cy.get('[data-cy="next-board-button"]')
-      .invoke('attr', 'title')
-      .then(title => {
-        nextDashboardName = title.toString();
+    dashboardNameGen,
+    columnEdgeValues,
+    switchIntervalEdgeValues,
+    dashboardNames
+} from '../fixtures/Dashboard'
+
+describe('Basic Dashboard CRUD', () => {
+    const dashboardName = dashboardNameGen()
+    const editDashboardName = dashboardNameGen('Edit')
+
+    beforeEach(() => {
+        cy.visit('/')
+        cy.login()
+    })
+
+    it('Logged user can add new dashboard', () => {
+        cy.addDashboard(dashboardName)
+    })
+
+    it('Logged user can choose dashboard', () => {
+        cy.addDashboard(dashboardName)
+        cy.contains('[data-cy="board-card"]', dashboardName)
+            .click()
+        cy.contains('[data-cy="navbar-title-header"]', dashboardName)
+            .should(
+                'exist'
+            )
```

Comment:
Is it necessary to put `'exist'` in separate line instead of one line `.should('exist')`?

Reply:
I've added new rule for this. Only 3 or more arguments will be required to be stated in multiple lines, otherwise there will be a warning.